### PR TITLE
Adding some text for Probot/Welcome-Bot for #FirstASFContribution

### PR DESCRIPTION
**Pull request by brucearctor**
Following up to make this concrete from discussion:
https://lists.apache.org/thread/8gs560t62srjkxq8n95m8ncg0xlj6shg

For this PR to do anything, we need to configure/install Welcome Bot ( https://github.com/behaviorbot/welcome )

via: https://github.com/apps/welcome/installations/new

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by brucearctor** [comment]
Added as potential reviews the people that commented in the ml thread ( https://lists.apache.org/thread/8gs560t62srjkxq8n95m8ncg0xlj6shg ).  

Let's either merge [ and install the bot ], discuss further [ if needed ], or just close this PR.  Any fine with me.  

Prefer to arrive at a resolution rather than this withering till stalebot closes it :-)

File: .github/config.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+firstPRMergeComment: |
+  Thank you for merging your first PR to the Beam community! 🙌
```
[the remark this answers is not in the document]
Comment (brucearctor) [reply .github/config.yml:2]: Makes sense.  Happy to update that text!  

I haven't been able to see our 6 week release calendar [ I think there was a google calendar at one point ] -- if that still exists, happy to link to it, any idea where that is?

[the remark this answers is not in the document]
Comment (brucearctor) [reply .github/config.yml:2]: https://calendar.google.com/calendar/u/0/embed?src=0p73sl034k80oob7seouanigd0@group.calendar.google.com

[the remark this answers is not in the document]
Comment (brucearctor) [reply .github/config.yml:2]: found it ... had searched website in the past.  It was referenced on list when someone volunteered to do the release management !

[the remark this answers is not in the document]
Comment (brucearctor) [reply .github/config.yml:2]: Unclear whether this is too long/detailed.  I think about and update the text for review

[the remark this answers is not in the document]
Comment (brucearctor) [reply .github/config.yml]: @robertwb -- how is the updated text?  Good to go?  Happy to make other changes/additions/etc.

**Comment by brucearctor** [comment]
@robertwb -- in addition to this code/PR merge.  We need to configure/install Welcome Bot ( https://github.com/behaviorbot/welcome )

via: https://github.com/apps/welcome/installations/new

Is that something you ( as PMC ) can do?

File: .github/config.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (brucearctor) [reply .github/config.yml]: 👍

**Comment by brucearctor** [comment]
I'll wait to merge the PR until we have the bot setup/in-place ... since without it, we could wind up with code in the repo that does nothing.

**Comment by damccorm** [comment]
stop reviewer notifications

**Pull request closed without merging**